include/irr/asset/IAssetManager.h

```diff
@@ -434,11 +434,15 @@ class IAssetManager : public core::IReferenceCounted
         /** Keeping assets around (by their pointers) helps a lot by letting the loaders retrieve them from the cache 
 		and not load cpu objects which have been loaded, converted to gpu resources and then would have been disposed of.
 		However each dummy object needs to have a GPU object associated with it in yet-another-cache for use when we convert CPU objects to GPU objects.*/
-        void convertAssetToEmptyCacheHandle(IAsset* _asset, core::smart_refctd_ptr<core::IReferenceCounted>&& _gpuObject, uint32_t referenceLevelsBelowToConvert=~0u)
+        void convertAssetToEmptyCacheHandle(IAsset* _asset, core::smart_refctd_ptr<core::IReferenceCounted>&& _gpuObject, uint32_t referenceLevelsBelowToConvert=0u)
         {
+            if (_asset->isDummyObjectForCacheAliasing)
```

Comment:
you can put this in front of the `grab` + insert into the CPU2GPU cache, but we need to call `convertToDummyObject` every single time (the function for the asset needs to handle multiple calls)

Reply:
i dont understand. What does it mean to handle multiple subsequent calls in this context?

Reply:
After I call `convertToDummyObject` on any `IAsset` I should be able to call `convertToDummyObject` again without any side-effects.

Reply:
basically I want this check inside the asset's `convertToDummyObject` method as well.